backend/interactive/cooldown-manager.js

```diff
@@ -219,9 +224,7 @@ async function mathUpdate(updateData) {
         }
 
         await mixplay.updateCooldownForControls(controlIds, newCooldown);
-        for (let controlId of controlIds) {
-            updateCooldownForControlId(controlId, newCooldown / 1000);
-        }
+        updateCooldownForControlId(controlId, newCooldown / 1000);
```

Comment:
I am wondering if there is still a bug here...
Currently this block looks like:
```js
for (let controlId of controlIds) {
        let remainingCooldown = getRemainingCooldownTime(controlId);
        let newCooldown;

        if (updateData.type === "add") {
            newCooldown = remainingCooldown + duration;
        } else {
            newCooldown = remainingCooldown - duration;
        }

        await mixplay.updateCooldownForControls(controlIds, newCooldown);
        updateCooldownForControlId(controlId, newCooldown / 1000);
}
```
It looks like we are looping through `controlIds` and updating ALL controls each time with `await mixplay.updateCooldownForControls(controlIds, newCooldown);`... is that correct? Seems we should only need to call `updateCooldownForControls` once or even not at all since we are updating the cooldown per control with `updateCooldownForControlId(controlId, newCooldown / 1000);` afterwards.

Reply:
Nope, definitely unintended. Will fix it!